label_studio/data_manager/serializers.py

```diff
@@ -202,11 +210,51 @@ def update(self, instance, validated_data):
             return instance
 
 
+class UpdatedByDMField(serializers.SerializerMethodField):
+    # TODO: get_updated_by implementation is weird, but we need to adhere schema to it
+    class Meta:
+        swagger_schema_fields = {
+            'type': openapi.TYPE_ARRAY,
+            'title': 'User IDs',
+            'description': 'User IDs who updated this task',
+            'items': {'type': openapi.TYPE_OBJECT, 'title': 'User IDs'},
+        }
+
+
+class AnnotatorsDMField(serializers.SerializerMethodField):
+    # TODO: get_updated_by implementation is weird, but we need to adhere schema to it
+    class Meta:
+        swagger_schema_fields = {
+            'type': openapi.TYPE_ARRAY,
+            'title': 'Annotators IDs',
+            'description': 'Annotators IDs who annotated this task',
+            'items': {'type': openapi.TYPE_INTEGER, 'title': 'User IDs'},
+        }
+
+
+class CompletedByDMSerializerWithGenericSchema(serializers.PrimaryKeyRelatedField):
+    # TODO: likely we need to remove full user details from GET /api/tasks/{id} as it non-secure and currently controlled by the export toggle
+    class Meta:
+        swagger_schema_fields = {
+            'type': openapi.TYPE_OBJECT,
+            'title': 'User details',
+            'description': 'User details who completed this annotation.',
+        }
+
+
+class AnnotationsDMFieldSerializer(AnnotationSerializer):
+    completed_by = CompletedByDMSerializerWithGenericSchema(required=False, queryset=User.objects.all())
+
+
+class AnnotationDraftDMFieldSerializer(AnnotationDraftSerializer):
+    result = AnnotationResultField(required=False)
+
+
 class DataManagerTaskSerializer(TaskSerializer):
-    predictions = serializers.SerializerMethodField(required=False, read_only=True)
-    annotations = AnnotationSerializer(required=False, many=True, default=[], read_only=True)
-    drafts = serializers.SerializerMethodField(required=False, read_only=True)
-    annotators = serializers.SerializerMethodField(required=False, read_only=True)
+    predictions = PredictionSerializer(required=False, many=True, default=[], read_only=True)
+    annotations = AnnotationsDMFieldSerializer(required=False, many=True, default=[], read_only=True)
+    drafts = AnnotationDraftDMFieldSerializer(required=False, many=True, default=[], read_only=True)
```

Comment:
It will break the call of `DataManagerTaskSerializer::get_drafts()` + `LseTaskSerializer::get_drafts()`

https://github.com/HumanSignal/label-studio/assets/501780/a17d6344-f77f-4291-9d6e-e16d1908272f

Reply:
Fixed for `predictions` and `drafts` - schemas were rewritten manually on top of `serializers.SerializerMethodField`